libraries/AP_OpticalFlow/OpticalFlow.cpp

```diff
@@ -59,7 +59,8 @@ const AP_Param::GroupInfo OpticalFlow::var_info[] = {
 
 // default constructor
 OpticalFlow::OpticalFlow(AP_AHRS_NavEKF &ahrs)
-    : _last_update_ms(0)
+    : _last_update_ms(0),
+      _ahrs(ahrs)
```

Comment:
comma should be together with `_ahrs` line. The reason for having this coding style here is so any future change in the initialization list needs to change only one line avoiding to keep a different pattern in the last line.